content/docs/app-developer-guide/run-an-app.md

```diff
@@ -69,58 +74,75 @@ The `launcher` will source any profile scripts (for `non-direct` processes) and
 
 If you would like to run the default process, you can simply run the app image without any additional arguments:
 
-```bash
+```
 docker run --rm -p 8080:8080 multi-process-app
 ```
+<!--+- "{{execute}}"+-->
 
 #### Default process type with additional arguments
 
 If you would like to pass additional arguments to the default process, you can run the app image with the arguments specified in the command:
 
-```bash
+```
 docker run --rm -p 8080:8080 multi-process-app --spring.profiles.active=mysql
 ```
+<!--+- "{{execute interrupt}}"+-->
 
 #### Non-default process-type
 
 To run a non-default process type, set the process type as the entrypoint for the run container:
 
-```bash
+```
 docker run --rm --entrypoint sys-info multi-process-app
 ```
+<!--+- "{{execute interrupt}}"+-->
 
 #### Non-default process-type with additional arguments
 
 You can pass additional arguments to a non-default process type:
 
-```bash
+```
 docker run --rm --entrypoint sys-info multi-process-app --spring.profiles.active=mysql
 ```
+<!--+- "{{execute interrupt}}"+-->
 
 #### User-provided shell process
 
 You can even override the buildpack-defined process types:
 
-```bash
+```
 docker run --rm --entrypoint launcher -it multi-process-app bash
+```
+<!--+- "{{execute interrupt}}"+-->
+
+Now let's exit this shell and run an alternative entrypoint - 
+```exit
```

Comment:
is this right? I just looks a little odd to me

Reply:
Yeah - you need to exit the docker shell to proceed with the next step.

Reply:
i didn't know ` ```exit` was legit

Reply:
Ah! Now I see what you meant! Yes - that is incorrect - good catch!